src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -829,6 +829,20 @@ def _cleanUpExternalStore(url):
         blobService, containerName, _ = AzureJobStore._extractBlobInfoFromUrl(urlparse.urlparse(url))
         blobService.delete_container(containerName)
 
+@needs_azure
+class InvalidAzureJobStoreTest(ToilTest):
+    def testInvalidJobStoreName(self):
+        from toil.jobStores.azureJobStore import AzureJobStore
+        self.assertRaises(ValueError,
+                          AzureJobStore.loadOrCreateJobStore,
+                          'us-west-2:a--b')
```

Comment:
us-west-2 is AWS nomenclature, correct. If so, using it for Azure job stores is at  best, misleading.  

The argument should be valid except for one aspect. If two aspects of it are wrong, you may not be asserting the right thing.